client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx

```diff
@@ -35,7 +35,7 @@ class SchemaVersions extends Root {
         return {
           id: schema.id,
           version: schema.version,
-          schema: JSON.stringify(JSON.parse(schema.schema), null, 2)
+          schema: "PROTOBUF" === formData.schemaType?latestSchemaVersion.schema :JSON.stringify(JSON.parse(latestSchemaVersion.schema), null, 2);
```

Comment:
This one will not work I think, there is no formdata on this page

Reply:
OOPS! I forgot to rename variable names. I will fix this and commit the changes after testing. Sorry!

Reply:
I've tested the code and fix the variable names.